src/player/Player.php

```diff
@@ -673,6 +673,10 @@ public function setUsingItem(bool $value) : void{
 		$this->networkPropertiesDirty = true;
 	}
 
+	public function allowForceEating () : bool{
```

Comment:
I'm not a fan of this name. It feels way too specific.

Reply:
> I'm not a fan of this name. It feels way too specific.

what is your idea?

Reply:
canPlayerConsume

canPlayerSnack
canPlayerChew
canPlayerDevour
isPlayerFamished

allowForceFeast
canFreelyEating

Reply:
Could `canEatAnytime` better ? It can't include `player` because it's linked to Human and not only Player.

Reply:
None of these address my original comment.
I'm thinking more along the lines of `canEat()`, which covers the creative case and hunger together.

Reply:
`canEat()` seems too general, but I too go with that.

Reply:
@ShockedPlot7560 Can I go with "canEatWithoutHunger"?

Reply:
I think I will go with "canEatWithoutHunger"